src/assets/js/search.js

```diff
@@ -88,6 +92,30 @@ function displaySearchResults(results) {
 
 }
 
+
+// Check if an element is currently scrollable
+function isScrollable(element) {
+    return element && element.clientHeight < element.scrollHeight;
+}
+
+// Ensure given child element is within the parent's visible scroll area
+function maintainScrollVisibility(activeElement, scrollParent) {
+    const { offsetHeight, offsetTop } = activeElement;
+    const { offsetHeight: parentOffsetHeight, scrollTop } = scrollParent;
+
+    const isAbove = offsetTop < scrollTop;
+    const isBelow = (offsetTop + offsetHeight) > (scrollTop + parentOffsetHeight);
+
+    if (isAbove) {
+        scrollParent.scrollTo(0, offsetTop);
+    }
+    else if (isBelow) {
```

Comment:
Why we have else if here?  Not just else.

Reply:
It is required to check if the element is below and then we need to scroll, removing that will change the scollbar position for visible elements also